Earthfile

```diff
@@ -18,7 +18,18 @@ markdown-check-fix:
 # check-spelling Check spelling in this repo inside a container.
 check-spelling:
     DO github.com/input-output-hk/catalyst-ci/earthly/cspell:v2.0.10+CHECK
- 
+
+# check-spelling Check spelling in this repo inside a container.
+spell-list-words:
```

Comment:
This is a utility test target which just lists out all words which fail spelling for a particular path.
It's how I created a custom dictionary that only effects the wasm code.
I left it in because its useful, and is not used unless specifically called but a user.